sdk/communication/azure-communication-rooms/azure/communication/rooms/_models/_models.py

```diff
@@ -90,14 +90,17 @@ class CommunicationRoom:
     :ivar valid_until: The timestamp from when the room can no longer be joined. The timestamp is
      in RFC3339 format: ``yyyy-MM-ddTHH:mm:ssZ``.
     :vartype valid_until: ~datetime.datetime
+    :ivar pstnDialOutEnabled: Set this flag to true if, at the time of the call, dial out to a PSTN number is enabled in a particular room. By default, this flag is set to false.
```

Comment:
This needs to use snake casing: `pstn_dial_out_enabled`.
This applies to everywhere in the API.